cooltools/api/expected.py

```diff
@@ -391,18 +365,16 @@ def make_diag_tables(
     return diag_tables
 
 
-def make_block_table(clr, regions1, regions2, clr_weight_name="weight", bad_bins=None):
+def make_block_table(clr, regions1, regions2, clr_weight_name="weight"):
     """
     Creates a table that characterizes a set of rectangular genomic blocks
     formed by combining regions from regions1 and regions2.
     For every block calculate its "area" in pixels ("n_total"), and calculate
     number of "valid" pixels in each block ("n_valid").
     "Valid" pixels exclude "bad" pixels, which in turn inferred from the balancing
-    weight column `clr_weight_name` or provided directly in the form of an array of
-    `bad_bins`.
+    weight column `clr_weight_name`.
 
-    Setting `clr_weight_name` and `bad_bins` to `None` yields 0 "bad" pixels per
-    block.
+    When `clr_weight_name` is `None`, no "bad" pixels are excluded.
```

Comment:
this is clear!

Reply:
or:
when `clr_weight_name` is `None`, raw data is used, and no "bad" pixels are bins are exclued.